pkg/config/validation/validation_test.go

```diff
@@ -1671,6 +1671,23 @@ func TestValidateTlsOptions(t *testing.T) {
 			},
 			"requires a private key", "not-a-cipher-suite",
 		},
+		{
+			"crl specified for SIMPLE TLS",
+			&networking.ServerTLSSettings{
+				Mode:  networking.ServerTLSSettings_SIMPLE,
+				CaCrl: "crl",
+			},
+			"CRL is not supported with SIMPLE TLS", "",
+		},
+		{
```

Comment:
This currently does not give the validation error as we return https://github.com/istio/istio/blob/28f23712a82ccf163c5ee8332c0e95fa7c62f670/pkg/config/validation/validation.go#L634. But I think we should atleast  add warnings. If folks agree, I can change that in a different PR